pkg/controller/controller.go

```diff
@@ -855,6 +855,7 @@ func (p *csiProvisioner) Delete(volume *v1.PersistentVolume) error {
 				},
 			})
 			if err != nil {
+				klog.Errorf("failed to get secretreference for volume %s: %v", volume.Name, err)
```

Comment:
Can we return the new error message instead?

Reply:
I changed to log err.Error(), if thats not you meant, please let me know